### PR TITLE
Ddf 2481 add windows commands to installation instructions

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Copy edits to quickstart and installation instructions.

#### Who is reviewing it?

@mcalcote @clockard @tbatie @garrettfreibott @emanns95 @mweser @spearskw

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@pklinef
@shaundmorris

#### How should this be tested?

Verify changes to content.

[DDF-2481](https://codice.atlassian.net/browse/DDF-2481)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc [hunk 1]
```diff
@@ -1,46 +1,74 @@
...
 setx PATH "%PATH%;%JAVA_HOME%\bin" /m
 ----
+====
 
 .*NIX
 [WARNING]
 ====
-Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
-`unlink /usr/bin/java`
+Unlink `<PATH_TO_JAVA>` if it is already linked to a previous version of the JRE:
```
Comment (emanns95) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: Should probably also be `JAVA_HOME`

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: ok

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc [hunk 2]
```diff
@@ -1,46 +1,74 @@
 
 This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
-Use this setup tutorial to test or try out the features of ${branding}. For a more thorough, secure installation, see <<_installing_${branding-lowercase},Installing ${branding}>>
+
+.QuickStart Uses
```
Comment (pklinef) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: Should quick start be two words?  It looks like QuickStart is a product.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: I'm on board. Two words.

Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: @ricklarsen did you push up the "QuickStart" change for this comment?

**Pull request merged**